AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts

```diff
@@ -1376,11 +1356,11 @@ describe('AddParticipantComponent edit mode no participants added', () => {
         expect(component.participantDetails).not.toBeNull();
         expect(component.participantDetails.username).toBeNull();
     });
-    it('should return JOH not found when role is Panel Member/Winger', () => {
-        role.setValue('Panel Member');
-        component.notFoundParticipant();
-        expect(component.errorJohAccountNotFound).toBeTruthy();
-    });
+    // it('should return JOH not found when role is Panel Member/Winger', () => {
```

Comment:
Does this need removing or adding back in?

Reply:
Forgot to remove these, good spot.